playground/webpack/parseMarkdown.js

```diff
@@ -16,9 +27,65 @@ module.exports = function loader() {
   });
   this.cacheable();
 
-  const stringyData = JSON.stringify(parseMarkdown(files), null, 2);
+  const data = parseMarkdown(files);
+
+  // Work around JSON.stringify() not supporting functions.
```

Comment:
I'm gonna have to ask you to explain this one in person later today

Reply:
It's a solid brainfuck. Let's chat and I can improve the comments